src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarm.java

```diff
@@ -488,7 +488,31 @@ public ListSuitesResult listSuites(String runArn) {
         return api.listSuites(request);
     }
 
+    /*
+     * get number of Unmetered devices based on upload type
+     */
+    public int getUnmeteredDevices(String os) {
```

Comment:
If you are adding documentation, please add proper JavaDocs per http://www.oracle.com/technetwork/articles/java/index-137868.html